### PR TITLE
Python: Fixes, example and docu improvements

### PR DESCRIPTION
**Pull request by FreyJo**

### CODE REVIEW

File: examples/acados_python/getting_started/minimal_example_closed_loop.py [hunk 1]
```diff
@@ -92,17 +92,18 @@
 ocp.constraints.x0 = x0
 ocp.constraints.idxbu = np.array([0])
 
-ocp.solver_options.qp_solver = 'PARTIAL_CONDENSING_HPIPM' # FULL_CONDENSING_QPOASES
+ocp.solver_options.qp_solver = 'PARTIAL_CONDENSING_HPIPM' # FULL_CONDENSING_Q_matPOASES
```
Comment (aghezz1) [inline examples/acados_python/getting_started/minimal_example_closed_loop.py:95]: Typo: FULL_CONDENSING_**Q_matPOASES**, should be FULL_CONDENSING_QPOASES

File: interfaces/acados_template/acados_template/acados_ocp_solver.py [hunk 2]
```diff
@@ -985,9 +985,9 @@ def __init__(self, acados_ocp, json_file='acados_ocp_nlp.json', simulink_opts=No
 
 
     def __get_pointers_solver(self):
-        """
-        Private function to get the pointers for solver
-        """
+        # """
+        # Private function to get the pointers for solver
+        # """
```
Comment (aghezz1) [inline interfaces/acados_template/acados_template/acados_ocp_solver.py:990]: Why do you comment the docstring?

**Review by aghezz1 — COMMENTED** [review]
@FreyJo please take a look to the two comments. Thanks!

File: interfaces/acados_template/acados_template/acados_ocp_solver.py (hunk 2 above)
Comment (FreyJo) [reply interfaces/acados_template/acados_template/acados_ocp_solver.py:990]: It was generated on the documentation page very prominently, which I didn't like.

**Review by aghezz1 — APPROVED** [review]

File: interfaces/acados_template/acados_template/acados_ocp_solver.py (hunk 2 above)
Comment (aghezz1) [reply interfaces/acados_template/acados_template/acados_ocp_solver.py:990]: Ok, I understand. We might consider to keep only line 989 as a standard comment with #, and delete the double quotes.

**Pull request merged**